packages/app-bridge/src/react/useDocumentPages.ts

```diff
@@ -33,29 +33,22 @@ export const useDocumentPages = (appBridge: AppBridgeTheme, documentId: number)
             setDocumentPages((previousState) => {
                 const isInitial = previousState === null;
 
+                const item = event.documentPageOrDocumentCategory as DocumentPage | DocumentCategory;
```

Comment:
It is not an `item` but `documentPageOrDocumentCategory`, let's refer to the correct entities as the function contains business logic